### DISCUSSION TITLE
Delete button

### DISCUSSION THREAD

**Discussion by arilofi** [community/community#136448] [Mobile]
Can you add delete file button

**Comment by Akash1134** [comment]
Hi @Aleu0091 , we're happy you're here! You are more likely to get a useful response if you are explicit giving a few more details might help someone give you a nudge in the right direction. Are you referring to adding this to the GitHub Mobile app? If so, please share the specifics and use cases you have top of mind. 

Thanks!

**Comment by ulyssecoispellier** [comment]
Heya you can go into your repo settings and delete it at the bottom of the page just read the indications that GitHub gives you and you should be all set ^^
